style.css

```diff
@@ -0,0 +1,528 @@
+@import url("https://fonts.cdnfonts.com/css/cocogoose");
+@import url("https://fonts.cdnfonts.com/css/Lato");
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;500;600;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Lato", sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+.nav-logo {
+  display: none;
+}
+
+h1 {
+  position: relative;
+  color: #e43c13;
+  line-height: 0.9;
+  font-size: 60px;
+  font-weight: bold;
+  padding: 1rem;
+}
+
+.bar1,
+.bar2,
+.bar3 {
+  width: 35px;
+  height: 5px;
+  background-color: #272a31;
+  margin: 6px 0;
+  z-index: 3;
+}
+
+.change .bar1 {
+  -webkit-transform: rotate(-45deg) translate(-9px, 6px);
+  transform: rotate(-45deg) translate(-9px, 6px);
+}
+
+.change .bar2 {
+  opacity: 0;
+}
+
+.change .bar3 {
+  -webkit-transform: rotate(45deg) translate(-8px, -8px);
+  transform: rotate(45deg) translate(-8px, -8px);
+}
+
+.nav {
+  justify-content: space-between;
+  padding-top: 10px;
+  overflow: hidden;
+  position: relative;
+  box-shadow: 1px 1px 3px #272a31;
+}
+
+#nav-links {
+  display: flex;
+  flex-direction: column;
+}
+
+#nav-links a {
+  color: rgb(68, 68, 68);
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  display: block;
+}
+
+.mobile-menu {
+  position: fixed !important;
+  z-index: 9999;
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.nav.mobile-menu {
+  justify-content: flex-start;
+  padding-top: 10px;
+  overflow: hidden;
+  position: relative;
+  box-shadow: 1px 1px 3px #272a31;
+}
+
+.intro-message {
+  margin-top: 50px;
+  position: relative;
+  color: #e43c13;
+  line-height: 1.1;
+  font-size: 35px;
+  padding-left: 1rem;
+}
+
+.intro-date {
+  font-size: 30px;
+  font-weight: 800;
+  margin-left: 1rem;
+}
+
+footer {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 3rem;
+  color: #d3d3d3;
+  background-color: whitesmoke;
```

Comment:
- [ ] Great job so far. Your design is very neat and clean 👏 🔥. However, I would suggest you adjust the `background- color` of the footer on the About page on the Desktop screen. **On the Design, it is very dark. Yours is somehow light grey**. Please change it to a dark color like `#302e2e` in order to resemble the Design you followed. Attention to detail is a great skill that every Developer should possess. **By paying attention to things like this, will make you outstanding** 😉
**NOTE** You will also **need to change the text color in the footer to light color so that they are easily visible** to the user.

Reply:
@BertrandConxy Thank you for that thorough observation.